### PR TITLE
task-59816-ui-user-create-account

### PR DESCRIPTION
**Pull request by nhunght**

### CODE REVIEW

File: app/assets/stylesheets/sessions.css.scss [hunk 1]
```diff
@@ -40,11 +39,57 @@
...
+  a {
+    float: right;
+  }
+}
+
+#sign-up {
+  margin: 0 150px;
+  h1 {
+    text-align: center;
+    font-family: "Open Sans",arial;
```
Comment (tqtrung09) [inline app/assets/stylesheets/sessions.css.scss]: have space after ,

File: app/assets/stylesheets/sessions.css.scss [hunk 2]
```diff
@@ -40,11 +39,57 @@
...
+
+  #sign-up-form {
+    margin: 0px;
+    padding: 25px;
+    background: none repeat scroll 0% 0% #F1F1F1;
+    border: 1px solid #E5E5E5;
+    float: right;
+    width: 300px;
+
+    .signup-button{
```
Comment (tqtrung09) [inline app/assets/stylesheets/sessions.css.scss]: have space before {

File: app/views/devise/registrations/new.html.erb [hunk 3]
```diff
@@ -1,41 +1,58 @@
...
 
-    <div class="field">
-      <%= f.label :email %><br/>
-      <%= f.email_field :email, autofocus: true, class: "form-border-bottom" %>
-    </div>
+        <div class="field">
+          <%= f.label :password %>
+          <% if @minimum_password_length %>
+            <em>
+              (<%= @minimum_password_length %> <%= t "devise.registrations.minimum" %>)
```
Comment (tqtrung09) [inline app/views/devise/registrations/new.html.erb]: remove space

**Comment by daodat18plus** [comment]
OK

**Comment by nguyenthaison** [comment]
oke

**Pull request merged**